eng/Versions.props

```diff
@@ -30,7 +30,7 @@
     <CodeStyleLayerCodeAnalysisVersion>3.3.1</CodeStyleLayerCodeAnalysisVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20067.1</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.5.0-beta3-20078-04</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.5.173-g9c5f5bc040</VisualStudioEditorPackagesVersion>
```

Comment:
Until 16.5 is released these changes should go into master-vs-deps

Reply:
Ah OK. I thought we merge on rpeview releases as well. Will retarget